docs/data/date-pickers/custom-field/custom-field.md

```diff
@@ -42,6 +42,13 @@ You can manually add an `endAdornment` if you want your range picker to look exa
 This adornment is purely decorative, the focus remains on the field when the picker is opened.
 :::
 
+### Change the format density
+
+You can make the format less dense using the `formatDensity` prop.
+Setting `formatDensity` to `"spacious"` will add a space before and after each `/`, `-` and `.` character.
+
+{{"demo": "SpaciousFieldFormat.js"}}
```

Comment:
If we add more examples, it would probably make sense to change the demo name to something like `FormatDensity` 🙈